lib/beaker-hostgenerator/generator.rb

```diff
@@ -22,7 +22,7 @@ class Generator
     def generate(layout, options)
       layout = prepare(layout)
       tokens = tokenize_layout(layout)
-      config = {}.deep_merge(BASE_CONFIG)
+      config = {}.deeper_merge(BASE_CONFIG)
```

Comment:
Should this be just
```suggestion
      config = BASE_CONFIG.dup
```

Or does `deeper_merge` also duplicate `CONFIG` within `BASE_CONFIG`?

Taking a deeper look: what is `consoleport`? In none of the beaker projects I can find any documentation. Is it something that only one provider (abs?) needs? Safe for nfs_server.

Reply:
Honestly I have no idea... `consoleport` sounds like it could be a PE-specific thing, but I 
don't know about `nfs_server`. I don't see them used anywhere in our projects... maybe it would make sense to remove both in a major release.

> Or does `deeper_merge` also duplicate `CONFIG` within `BASE_CONFIG`?

That seems to be the case, object IDs are different with `deeper_merge`:
```
[20] pry(#<BeakerHostGenerator::Generator>)> BASE_CONFIG.dup["CONFIG"].object_id == BASE_CONFIG["CONFIG"].object_id
=> true
[21] pry(#<BeakerHostGenerator::Generator>)> {}.deeper_merge(BASE_CONFIG)["CONFIG"].object_id == BASE_CONFIG["CONFIG"].object_id
=> false
```

Reply:
Looks like consoleport was introduced [here](https://github.com/voxpupuli/beaker/commit/7492b5b6415fff2e4d525a3ccb5b76537d45179d) and nfs_server [here](https://github.com/voxpupuli/beaker/commit/f6523410f3bb57e4421c9856f96d6798dbc96adf). Both are PE-specific. I think we should do a major version where we drop them. It's adding a lot of noise for something that's barely if at all used.